src/ResourceGraph/ResourceGraph/Cmdlets/SearchAzureRmGraph.cs

```diff
@@ -79,7 +84,7 @@ public string[] Subscription
         /// <summary>
         /// Gets or sets the management groups.
         /// </summary>
-        [Parameter(Mandatory = false, ParameterSetName = TenantParameterSet, HelpMessage = "Management group(s) to run query against")]
+        [Parameter(Mandatory = false, ParameterSetName = ManagementGroupParameterSet, HelpMessage = "Management group(s) to run query against")]
```

Comment:
no need to add parameter set for optional parameter

Reply:
The logic here was to prevent users passing -ManagementGroup, -Subscription and -OnTenantScope parameters at the same time. It should be either netiher of these params or just 1 specified.

Would you recommend to implement custom handling of accepted parameters combination?

Reply:
one way is to keep these parameter sets and make the switch parameter mandatory, however implement custom parameter validation is also fine and I suggest to clarify that in parameter description